interfaces/builtin/daemon_notify.go

```diff
@@ -44,6 +44,9 @@ const daemonNotifyConnectedPlugAppArmorTemplate = `
 
 # Allow using systemd-notify in shell scripts.
 /{,usr/}bin/systemd-notify ixr,
+
+capability sys_admin,
```

Comment:
I'm sceptical about those being actually required. Can you justify this somehow? What was the denial you had otherwise observed?

Reply:
Yes see my other comment or the LP bug but these denials come from the `systemd-notify` binary directly.